hedgehog.cabal

```diff
@@ -34,15 +34,17 @@ source-repository head
 library
   build-depends:
       base                            >= 3          && < 5
-    , base64-bytestring               >= 1.0        && < 1.1
+    , ansi-terminal                   >= 0.6        && < 0.7
```

Comment:
I'm not too worried about `ansi-terminal` as a dependency, it's absolutely micro and only depends libraries which already ship with GHC itself (`unix` / `Win32` depending on the platform).